pageserver/src/tenant.rs

```diff
@@ -4089,6 +4069,13 @@ mod tests {
             .create_test_timeline(TIMELINE_ID, Lsn(0x10), DEFAULT_PG_VERSION, &ctx)
             .await?;
         drop(tline);
+        // so that all uploads finish & we can call harness.try_load() below again
+        tenant
+            .shutdown(Default::default(), true)
+            .instrument(info_span!("test_shutdown", tenant_id=%tenant.tenant_id))
+            .await
+            .ok()
+            .unwrap();
```

Comment:
This is to drop the non-Debug completion barrier. Before #4916 we don't really have anything to debug, but of course, it could have a dummy Debug impl.

Reply:
resolving this, in case someone comes looking.